thirdparty/kyaml/runfn/runfn.go

```diff
@@ -386,25 +386,14 @@ func getUIDGID(asCurrentUser bool, currentUser currentUserFunc) (string, error)
 	return fmt.Sprintf("%s:%s", u.Uid, u.Gid), nil
 }
 
-// etFunctionConfig returns yaml representation of functionConfig that can
+// getFunctionConfig returns yaml representation of functionConfig that can
 // be provided to a function as input.
 func (r *RunFns) getFunctionConfig() (*yaml.RNode, error) {
-	if r.FnConfigPath == "" {
-		return nil, fmt.Errorf("function config file path must not be empty")
-	}
-	f, err := os.Open(r.FnConfigPath)
-	if err != nil {
-		return nil, fmt.Errorf("missing function config '%s': %w", r.FnConfigPath, err)
-	}
-	reader := kio.ByteReader{Reader: f}
-	nodes, err := reader.Read()
+	node, err := v1alpha2.ValidateFnConfigPath("", r.FnConfigPath)
```

Comment:
Just return `v1alpha2.ValidateFnConfigPath("", r.FnConfigPath)` or remove this thin wrapper.

Reply:
done